DirectProgramming/DPC++FPGA/ReferenceDesigns/qri/sample.json

```diff
@@ -0,0 +1,69 @@
+{
+  "guid": "A42A1772-2448-4f67-AB28-4080FE47F9C1",
```

Comment:
Please confirm you generated a unique GUID for this sample

Reply:
Yes